gcp-connector-util/gcp-cups-connector-util.go

```diff
@@ -174,7 +174,7 @@ func getGCP(config *lib.Config) (*gcp.GoogleCloudPrint, error) {
 	return gcp.NewGoogleCloudPrint(config.GCPBaseURL, config.RobotRefreshToken,
 		config.UserRefreshToken, config.ProxyName, config.GCPOAuthClientID,
 		config.GCPOAuthClientSecret, config.GCPOAuthAuthURL, config.GCPOAuthTokenURL,
-		0, nil)
+		0, nil, false)
```

Comment:
FCM does not require any configuration so it may be a fit in util class, gcp-use-fcm flag is just a run time signal for connector to chose b/w xmpp and fcm channels and so it should ideally belong to connector.